### PR TITLE
Update sdk (track 1)

### PR DESCRIPTION
**Pull request by yibirnba**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by simanor — APPROVED** [review]

File: sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj [hunk 1]
```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.SecurityCenter</PackageId>
     <Description>Provides developers with libraries for the updated Azure Security Center platform under Azure Resource manager to view and manage security posture in and outside Azure.</Description>
-    <Version>3.0.0</Version>
+    <Version>3.0.1</Version>
```
Comment (fengzhou-msft) [inline sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj:9]: Please bump the version to `4.0.0`.

File: sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
     <AssemblyName>Microsoft.Azure.Management.SecurityCenter</AssemblyName>
     <PackageTags>management;security center;security;IoT security;</PackageTags>
-    <PackageReleaseNotes>Updated SDK client. Bumpes settings API Version</PackageReleaseNotes>
+    <PackageReleaseNotes>Updated SDK client.</PackageReleaseNotes>
```
Comment (fengzhou-msft) [inline sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj:12]: Add notes about all the breaking changes.

File: sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/AlertsOperations.cs [hunk 3]
```diff
@@ -1073,7 +1073,7 @@ public async Task<AzureOperationResponse<Alert>> GetSubscriptionLevelWithHttpMes
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<Alert>> GetResourceGroupLevelWithHttpMessagesAsync(string alertName, string resourceGroupName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<Alert>> GetResourceGroupLevelWithHttpMessagesAsync(string resourceGroupName, string alertName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
[the remark this answers is not in the document]
Comment (yibirnba) [reply sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/AlertsOperations.cs:1076]: The change in parameters order was made after a validation failed in swagger (as the former order wasn't aligned with the url order "should be kept in the same order as they present in the path."). Also, track 2 uses this version and generated the new order of the parameters. I'll keep it as breaking change to be aligned with swagger validations and with track 2 methods. I'll add notes and bump major version

File: sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/AssessmentsMetadataOperationsExtensions.cs [hunk 4]
```diff
@@ -27,7 +27,7 @@ public static partial class AssessmentsMetadataOperationsExtensions
             /// <param name='operations'>
             /// The operations group for this extension method.
             /// </param>
-            public static IPage<SecurityAssessmentMetadata> List(this IAssessmentsMetadataOperations operations)
+            public static IPage<SecurityAssessmentMetadataResponse> List(this IAssessmentsMetadataOperations operations)
```
[the remark this answers is not in the document]
Comment (MorBrisker) [reply sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/AssessmentsMetadataOperationsExtensions.cs:30]: Hi @fengzhou-msft , this is indeed a breaking change that was added in this [PR](https://github.com/Azure/azure-rest-api-specs/pull/15196) for a new API version

[the remark this answers is not in the document]
Comment (yibirnba) [reply sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/AssessmentsMetadataOperationsExtensions.cs:30]: Track 2 generated files also include the new naming

File: sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj (hunk 2 above)
Comment (yibirnba) [reply sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj]: Added. Can you please check if the comments are informative enough?

File: sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj (hunk 1 above)
Comment (yibirnba) [reply sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Microsoft.Azure.Management.SecurityCenter.csproj]: Bumped

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**